src/librustdoc/lib.rs

```diff
@@ -597,7 +599,10 @@ fn usage(argv0: &str) {
     }
     println!("{}", options.usage(&format!("{} [options] <input>", argv0)));
     println!("    @path               Read newline separated options from `path`\n");
-    println!("More information available at https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html")
+    println!(
+        "More information available at https://doc.rust-lang.org/{}/rustdoc/what-is-rustdoc.html",
```

Comment:
Hmm, I just realized that unfortunately links like this will look like `doc.rust-lang.org//rustdoc/...` on stable. Maybe a better approach would be a function that returns `https://doc.rust-lang.org/nightly`, `https://doc.rust-lang.org/1.50.0`? Not sure if you want to do that here or in a follow-up, but I think we should fix it because it looks weird to users.

Reply:
It already does that :) it will only have `//` links if you build a custom rustdoc *and* change `channel` to something it doesn't recognize, which seems a like a really uncommon edge case.

Reply:
Here's with `channel = "xxx"`:
```
More information available at https://doc.rust-lang.org//rustdoc/what-is-rustdoc.html
```
and with `channel = "stable"`:
```
More information available at https://doc.rust-lang.org/1.53.0/rustdoc/what-is-rustdoc.html
```

Reply:
Ah, I forgot about that! That seems fine then.